server/service/src/sync/test/test_data/goods_received.rs

```diff
@@ -0,0 +1,60 @@
+use crate::sync::test::TestSyncIncomingRecord;
+use chrono::NaiveDate;
+use repository::goods_received_row::{
+    GoodsReceivedRow, GoodsReceivedRowDelete, GoodsReceivedStatus,
+};
+
+const TABLE_NAME: &str = "goods_received";
+
+const GOODS_RECEIVED: (&str, &str) = (
+    "GOODS_RECEIVED",
+    r#"{
+        "ID": "3486239A597646B2B7259D91A24988E8",
+        "budget_ID": "",
+        "comment": "",
+        "donor_id": "",
+        "entry_date": "2025-07-24",
+        "linked_transaction_ID": "",
```

Comment:
Same here, my mistake - this will need a value in it I believe to satisfy the remote reference constraint.

```suggestion
"linked_transaction_ID": "622C9D65F5124BA5B11DF55B6FB1627B",
```